app/views/api/v2/app_visitor_configs/_show.json.jbuilder

```diff
@@ -0,0 +1,10 @@
+json.splits active_splits do |split|
+  json.name split.name
+  json.weights split.registry
+  json.feature_gate split.feature_gate?
+end
```

Comment:
this is part of the meaningful changes in the PR: now this is an array instead of a map